linux_docker_resources/Dockerfile

```
@@ -122,6 +122,15 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
   uncrustify \
   yamllint
 
+# Configure proposed source, and install python3-flake8 from it
+RUN if test \( ${UBUNTU_DISTRO} = noble \); then\
+    printf "deb http://archive.ubuntu.com/ubuntu/ "$(lsb_release -cs)"-proposed restricted main multiverse universe" >> /etc/apt/sources.list.d/ubuntu-$(lsb_release -cs)-proposed.list && \
+    printf "Package: *\nPin: release a="$(lsb_release -cs)"-proposed\nPin-Priority: -10" >> /etc/apt/preferences.d/proposed-updates && \
```

```suggestion
    printf "deb http://archive.ubuntu.com/ubuntu/ "$(lsb_release -cs)"-proposed restricted main multiverse universe\n" >> /etc/apt/sources.list.d/ubuntu-$(lsb_release -cs)-proposed.list && \
    printf "Package: *\nPin: release a="$(lsb_release -cs)"-proposed\nPin-Priority: -10\n" >> /etc/apt/preferences.d/proposed-updates && \
```

I noticed this when reading job logs. Please add trailing newlines when using printf.